pkg/monitoring/metrics/virt-controller/vmstats_collector.go

```diff
@@ -185,6 +193,7 @@ func vmStatsCollectorCallback() []operatormetrics.CollectorResult {
 	results = append(results, CollectVMsInfo(vms)...)
 	results = append(results, CollectResourceRequestsAndLimits(vms)...)
 	results = append(results, reportVmsStats(vms)...)
+	results = append(results, CollectVMCreatedTime(vms)...)
```

Comment:
```suggestion
    results = append(results, collectVMCreationTimestamp(vms)...)
```
I think `collectVMCreationTimestamp` is better and not exported :)

Reply:
fixed thanks